operations/workplace/people/working-at-mattermost/onboarding/engineer-onboarding.md

```diff
@@ -49,6 +49,7 @@ This is the general timeline and expectations for new engineers starting at Matt
   * Go over core values and Leadership Principles with dev lead
 * Work on 1 - 3 small tickets
 * Introduction at the [Customer Obsession Meeting](https://handbook.mattermost.com/operations/operations/company-cadence#customer-obsession-meeting-aka-com)
+* Review the [Engineering Titles/Levels](https://docs.google.com/spreadsheets/d/11pG9OqsLTltVylGRDBUOcTa65kkHd_EtRiPBz8Mnnpk/edit#gid=2064417787) sheet to understand expectations and responsibilities for your level
```

Comment:
Not sure if this should be second week or later on - open to suggestions on that

Reply:
I might be inclined to stick it in the Weeks 3/4 -- I sometimes find there's already so much to cover in Weeks 1 and 2. It's also hard not to chat about this without also chatting about overall performance, and it might be more useful to have a full two weeks in (both from the manager and the employee's perspective).